.build/build-resolver.xml

```diff
@@ -56,6 +56,7 @@
         <resolver:remoterepos id="all">
             <remoterepo id="resolver-central" url="${artifact.remoteRepository.central}"/>
             <remoterepo id="resolver-apache" url="${artifact.remoteRepository.apache}"/>
+            <remoterepo id="resolver-apache-snapshots" url="${artifact.remoteRepository.apache-snaphot}" snapshots="true"/>
```

Comment:
I vaguely remember someone being opposed to adding snapshots when I did the same, but tbh I can not recall the details.

Reply:
yes yes! that was me. this is in a separate commit and will be removed before the merge.

**OR** we find a way to allow the snapshot repository in between releases… more thought needed…